bin/install-if-no-packages.js

```diff
@@ -2,10 +2,11 @@ const spawnSync = require( 'child_process' ).spawnSync;
 const fs = require( 'fs' );
 
 if ( ! fs.existsSync( 'node_modules' ) ) {
-	console.log( 'No "node_modules" present, installing dependencies...' );
+	console.log( 'No "node_modules" present, installing dependencies…' );
 	const installResult = spawnSync( 'npm', [ 'ci' ], {
 		shell: true,
 		stdio: 'inherit',
+		env: Object.assign( { PUPPETEER_SKIP_CHROMIUM_DOWNLOAD: 'true' }, process.env ),
 	} ).status;
```

Comment:
Here I see the same error as I fixed in #39009 for `install-if-deps-outdated` -- wrong type of `installResult` and suboptimal error message. I'll push a quick fix to this branch.

Reply:
Done in https://github.com/Automattic/wp-calypso/pull/38900/commits/ae81a1be7cd8190427e03d936557ea80d1ea191c